website/content/docs/secrets/aws.mdx

```diff
@@ -305,6 +302,56 @@ boundary policies that you wish to ensure that Vault uses. This policy will
 ensure that Vault uses one of the permissions boundaries specified (not all of
 them).
 
+### Plugin Workload Identity Federation (WIF)
+
+<EnterpriseAlert product="vault" />
+
+The AWS secrets engine supports the Plugin WIF workflow, and has a source of identity called
+a plugin identity token. The plugin identity token is a JWT that is internally signed by Vault's 
+[plugin identity token issuer](/vault/api-docs/secret/identity/tokens#read-plugin-workload-identity-issuer-s-openid-configuration). 
+Given a trust relationship is configured between Vault and AWS via [Web Identity Federation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_oidc.html),
+the secrets engine can exchange its identity token for short-lived STS credentials needed to
+perform its actions. This enables the secrets engine to operate without needing sensitive
+IAM security credentials as a part of its configuration.
+
+To configure the secrets engine to use plugin WIF, you must first:
+
+1. Ensure that Vault [openid-configuration](/vault/api-docs/secret/identity/tokens#read-plugin-identity-token-issuer-s-openid-configuration)
+   and [public JWKS](/vault/api-docs/secret/identity/tokens#read-plugin-identity-token-issuer-s-public-jwks) 
+   APIs are network-reachable by AWS
+   
+   An API proxy or gateway can be used to limit Vault API exposure depending on your environment.  
+
+1. Create an [IAM OIDC identity provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html) in AWS
+   
+   The provider URL must point at Vault's [plugin identity token issuer](/vault/api-docs/secret/identity/tokens#read-plugin-workload-identity-issuer-s-openid-configuration). 
+   Remove the `/.well-known/openid-configuration` suffix of the URL and provide it like 
+   `https://host:port/v1/identity/oidc/plugins`.
+   
+   The audience should uniquely identify the recipient of the plugin identity token. In AWS, 
+   the recipient is the identity provider. We recommend setting this to the `host:port/v1/identity/oidc/plugins`
+   portion of the provider URL since it's enforced as unique per configured identity provider.
```

Comment:
```suggestion
1. Create an
   [IAM OIDC identity provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html)
   in AWS.
   1. The provider URL **must** point at your [Vault plugin identity token issuer](/vault/api-docs/secret/identity/tokens#read-plugin-workload-identity-issuer-s-openid-configuration) with the
   `/.well-known/openid-configuration` suffix removed. For example: 
   `https://host:port/v1/identity/oidc/plugins`.
   1. The audience should uniquely identify the recipient of the plugin identity
   token. In AWS, the recipient is the identity provider. We recommend using
   the `host:port/v1/identity/oidc/plugins` portion of the provider URL as your
   recipient since it will be unique for each configured identity provider.
```

Reply:
Thanks!